.github/workflows/webhook.yml

```
@@ -0,0 +1,154 @@
+                "workflow_url": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}",
+                "timestamp": "${{ github.event.workflow_run.created_at }}"
+              }
+            }
+
+      - name: n8n 웹훅 전송
+        env:
+          WEBHOOK_URL: ${{ secrets.N8N_WEBHOOK_URL }}
+        run: |
+          if [ -n "$WEBHOOK_URL" ]; then
+            echo "웹훅 전송 중..."
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)